scripts/globals/beastmentreasure.lua

```diff
@@ -0,0 +1,394 @@
+------------------------------------
+-- Beastmen Treasure side quests
+-- https://ffxiclopedia.fandom.com/wiki/Beastmen_Treasure
+------------------------------------
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/weather')
+------------------------------------
+dsp = dsp or {}
+dsp.beastmentreasure = dsp.beastmentreasure or {}
+
+local PICKAXE = 605
+
+local zoneData = {
+    -- Yuhtunga Jungle (Elshimo Lowlands)
+    [123] =
+    {
+        statusvar = 'BMT_Lowlands_Status',
```

Comment:
Does this really new 2 variables? Can't we just check if a QM is set as a variable.